src/services/search.service.js

```diff
@@ -0,0 +1,110 @@
+import { Op } from 'sequelize';
+import TripService from './trip.service';
+import RequestService from './request.service';
+import UserService from './user.service';
+
+/**
+ *
+ *
+ * @class searchService
+ */
+class SearchService {
+  /**
+   *
+   *
+   * @static
+   * @param {req} term
+   * @returns {response} @memberof SearchService
+   */
+  static async getLocationIdQuery(term) {
+    const locations = await TripService.findAllLocations({
+      [Op.or]: [{ country: { [Op.iLike]: `%${term}%` } }, { city: { [Op.iLike]: `%${term}%` } }]
+    });
+    const ids = locations.map((location) => {
```

Comment:
is this variable has meaning? would be better if you find a descriptive variable name